Makefile

```diff
@@ -244,32 +254,33 @@ kind-install-ic:
 
 .PHONY: kind-install-cilium
 kind-install-cilium:
-	kind load docker-image --name kube-ovn $(REGISTRY)/kube-ovn:$(RELEASE_TAG)
-	ENABLE_SSL=true ENABLE_LB=false ENABLE_NP=false dist/images/install.sh
-	kubectl describe no
-	kubectl taint node kube-ovn-control-plane node-role.kubernetes.io/master:NoSchedule-
+	$(eval KUBERNETES_SERVICE_HOST = $(shell kubectl get nodes kube-ovn-control-plane -o jsonpath='{.status.addresses[0].address}'))
+	if ! docker images --format "{{.Repository}}:{{.Tag}}" | grep -qw "^$(CILIUM_IMAGE_REPO):v$(CILIUM_VERSION)$$"; then \
+		docker pull "$(CILIUM_IMAGE_REPO):v$(CILIUM_VERSION)"; \
+	fi
+	kind load docker-image --name kube-ovn "$(CILIUM_IMAGE_REPO):v$(CILIUM_VERSION)"
 	kubectl apply -f yamls/chaining.yaml
-	kind get nodes --name kube-ovn | while read node; do \
-    	docker exec $$node mv /etc/cni/net.d/01-kube-ovn.conflist /etc/cni/net.d/10-kube-ovn.conflist; \
-	done
-	$(eval CONTROLLERIP = $(shell kubectl get nodes kube-ovn-control-plane -ojsonpath='{.status.addresses[0].address}'))
 	helm repo add cilium https://helm.cilium.io/
-	helm install cilium cilium/cilium --version 1.10.5 \
+	helm install cilium cilium/cilium \
+		--version $(CILIUM_VERSION) \
 		--namespace=kube-system \
-		--set cni.chainingMode=generic-veth \
-		--set cni.customConf=true \
-		--set cni.configMap=cni-configuration \
+		--set k8sServiceHost=$(KUBERNETES_SERVICE_HOST) \
+		--set k8sServicePort=6443 \
 		--set tunnel=disabled \
 		--set enableIPv4Masquerade=false \
 		--set enableIdentityMark=false \
 		--set kubeProxyReplacement=strict \
-		--set k8sServiceHost=$(CONTROLLERIP) \
-		--set k8sServicePort=6443
-	kubectl -n kube-system delete ds kube-proxy
-	kubectl -n kube-system delete cm kube-proxy
+		--set cni.chainingMode=generic-veth \
+		--set cni.customConf=true \
+		--set cni.configMap=cni-configuration
+	kubectl -n kube-system rollout status ds cilium --timeout 300s
```

Comment:
status check for cilium pods should be moved to the end of the process where all CNIs are installed.
Meanwhile, before that check, these pods may need to be rebooted.